litestar/cli/commands/core.py

```diff
@@ -120,15 +124,32 @@ def info_command(app: Litestar) -> None:
 
 
 @command(name="run")
-@option("-r", "--reload", help="Reload server on changes", default=False, is_flag=True)
-@option("-R", "--reload-dir", help="Directories to watch for file changes", multiple=True)
+@option("-r", "--reload", help="Reload server on changes", default=False, is_flag=True, envvar="LITESTAR_RELOAD")
 @option(
-    "-I", "--reload-include", help="Glob patterns for files to include when watching for file changes", multiple=True
+    "-R",
+    "--reload-dir",
```

Comment:
I guess if we can do it like this we should add an option `reload-dirs` and deprecate this in 3.0?
idk.

Reply:
Am I right in thinking this means now we can use either `--reload-dir=./dir/a,./dir/b` and `--reload-dir=./dir/a --reload-dir=./dir/b` at the cli?

Reply:
😕 not enough brain wrinkles for me to process